provider/identity_provider_resource.go

```diff
@@ -139,10 +172,16 @@ func (r *IdentityProviderResource) Create(ctx context.Context,
 		)
 		return
 	}
+	// handle mapping_method
+	if state.MappingMethod.Unknown || state.MappingMethod.Null {
+		// Set default Value
+		state.MappingMethod.Value = defaultMappingMethod
```

Comment:
you can't do it
this is an optional only attribute
If you will set it here you change the value of that attribute.... you have 2 options: 
 1. change it to be also computed
 2. no setting the value if it wasn't set